packages/fiori/src/ShellBar.hbs

```diff
@@ -107,6 +109,7 @@
 					icon="sap-icon://search"
 					data-ui5-text="Search"
 					data-ui5-notifications-count="{{notificationsCount}}"
+					data-ui5-stable="search-button"
```

Comment:
Technically this button doesn't execute search - it toggles the search to be shown or not shown.
```suggestion
					data-ui5-stable="toggle-search"
```